test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -76,7 +76,7 @@ def run_ort(ort_sess, input):
     input, _ = torch.jit._flatten(input_copy)
     inputs = [to_numpy(inp) for inp in input]
 
-    ort_inputs = dict((ort_sess.get_inputs()[i].name, input) for i, input in enumerate(inputs))
+    ort_inputs = dict((ort_input.name, inputs[i]) for i, ort_input in enumerate(ort_sess.get_inputs()))
```

Comment:
This might be unsafe for cases where inputs in the middle happens to be unused, while this takes the first `k` inputs.